python_utils/src/elasticai/testing/hw_test.py

```diff
@@ -0,0 +1,19 @@
+import shutil as sh
+import argparse
+import time
+
+if __name__ == "__main__":
+    """example call 
+    python -m elasticai.testing.hw_test -d /media/jan/RPI-RP2/ -p /dev/ttyACM0 -t ../build/env5_rev2_debug/test/hardware/Esp/HWTestAskEspForOkViaPico.uf2"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-p", "--port")
+    parser.add_argument("-t", "--test")
+    parser.add_argument("-d", "--dir")
+    args = parser.parse_args()
```

Comment:
Why don't you use the argparse library to check for the correct arguemnts and provide help?

```suggestion
    """running integration tests
    Help:
        execute `python3 ./hw_tst.py --help`
    """
    parser = argparse.ArgumentParser(prog="HW-Test", description="Hardware-Test Runner")
    parser.add_argument("-p", "--port", requierd=True, help="Device COM-Port")
    parser.add_argument("-t", "--test", requierd=True, help="Path to hardware-tests executables")
    parser.add_argument("-d", "--dir", requierd=True, help="??")
    args = parser.parse_args()
```